rdrf/rdrf/features/steps.py

```diff
@@ -41,9 +41,58 @@ def scroll_to(element):
      scroll_to_y(y)
      print("scrolled to %s at (0,Y) = %s" % (moniker(element), y))
      return y
+
+
+def scroll_to_multisection_cde(section, cde, item=1):
```

Comment:
Can we rename scroll_to_multisection_cde to find_multi_section_cde (or similar) and remove the scroll_to call. I think we should separate those two functions - finding things and clicking on them. Both can be a bit tedious to debug in aloe/selenium.

Related: In case it was missed: there is a utils.click method that scrolls to items before clicking.

I realise there are not in the PR (earlier commits) but if the scroll_to() methods are needed can we move them to utils.py, it might also be a good idea to add a note about why they are needed / or under what conditions the other scroll/click methods don't work.